### PR TITLE
Full path to vignettes

### PR DESCRIPTION
**Pull request by d-golzato**
Changed relative path to full path in README.md to fix links in pkgdown

### CODE REVIEW

**Review by lwaldron — COMMENTED** [review]
Looks good to me and should close #36 . @schifferl take a look and merge unless you have another idea.

File: README.md [hunk 1]
```diff
@@ -29,16 +29,16 @@ Visit `localhost` in your browser.
 
 ### R Vignettes
 
-* [Create datasets for machine learning](vignettes/MLdatasets.Rmd)
-* [Exploration of the liver cirrhosis dataset](vignettes/explorecirrhosis.Rmd)
-* [Meta-analysis of age-related microbial species using cMD3](vignettes/Age_metaanalysis_vignette.Rmd)
-* [Meta-analysis of sex-related microbial species using cMD3](vignettes/Sex_metaanalysis_vignette.Rmd)
-* [NUI Galway Metagenomics Workshop](vignettes/NUI-Galway-Metagenomics-Workshop.Rmd)
+* [Create datasets for machine learning](https://github.com/waldronlab/curatedMetagenomicAnalyses/blob/vignettes/MLdatasets.Rmd)
```
Comment (schifferl) [inline README.md:32]:
```suggestion
* [Create datasets for machine learning](https://waldronlab.io/curatedMetagenomicAnalyses/articles/MLdatasets.html)
```

**Review by schifferl — CHANGES_REQUESTED** [review]
Hi @d-golzato, the approach was correct, but the links were a little off. I made suggestions with the correct links that can be committed and then we will merge this pull request.

**Comment by d-golzato** [comment]
Great, thanks @schifferl! Should I merge this pull request by myself?

**Review by schifferl — APPROVED** [review]

**Comment by schifferl** [comment]
Good work @d-golzato, merging and closing now.

**Pull request merged**